components/hypercert/evaluate-drawer.tsx

```diff
@@ -67,6 +66,13 @@ export function EvaluateDrawer({ hypercert }: { hypercert: HypercertFull }) {
     });
   const [comments, setComments] = useState<string>("");
   const [uid, setUid] = useState<string>();
+  const [Tags, setTags] = useState<any>(null);
```

Comment:
Can this be an array of strings or null?
```suggestion
  const [Tags, setTags] = useState<string[] | null>(null);
```

Reply:
Its actually a placeholder for a dynamic import of a Tag component, not a list of tags (see useEffect on line 70)